MemeifyMe/app/src/main/res/layout/activity_main.xml

```diff
@@ -9,9 +9,52 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     tools:context=".MainActivity">
 
+
```

Comment:
As a general rule, please remember to run Code > Re-arrange code in order to keep everything organized and consistent. Thanks!